<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transactions_report_dialog</class>
 <widget class="QDialog" name="transactions_report_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbLastBought">
       <property name="text">
        <string>Last bought:</string>
       </property>
       <property name="buddy">
        <cstring>dtLastBought</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dtLastBought">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbLastSold">
       <property name="text">
        <string>Last sold:</string>
       </property>
       <property name="buddy">
        <cstring>dtLastSold</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dtLastSold">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>72</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tbReport">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbUnitsBought">
       <property name="text">
        <string>Units bought:</string>
       </property>
       <property name="buddy">
        <cstring>sbUnitsBought</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="localized_spin_box" name="sbUnitsBought">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> units</string>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="localized_spin_box" name="sbTotalBought">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> gp</string>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbUnitsSold">
       <property name="text">
        <string>Units sold:</string>
       </property>
       <property name="buddy">
        <cstring>sbUnitsSold</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="localized_spin_box" name="sbUnitsSold">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> units</string>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="localized_spin_box" name="sbTotalSold">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> gp</string>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbDiff">
       <property name="text">
        <string>Difference:</string>
       </property>
       <property name="buddy">
        <cstring>sbDiff</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="localized_spin_box" name="sbDiff">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> gp</string>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>localized_spin_box</class>
   <extends>QSpinBox</extends>
   <header>localized_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dtLastBought</tabstop>
  <tabstop>dtLastSold</tabstop>
  <tabstop>tbReport</tabstop>
  <tabstop>sbUnitsBought</tabstop>
  <tabstop>sbTotalBought</tabstop>
  <tabstop>sbUnitsSold</tabstop>
  <tabstop>sbTotalSold</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
